rfcs/convergence/ending-ie11-support.md

```diff
@@ -0,0 +1,78 @@
+# RFC: Ending support for IE11 with Fluent v9
+
+@ling1726
+
+## Summary
+
+Fluent v9 will officially no longer support IE11.
+
+[Microsoft has officially announced a deadline to end IE11 support for M365 Apps](https://techcommunity.microsoft.com/t5/microsoft-365-blog/microsoft-365-apps-say-farewell-to-internet-explorer-11-and/ba-p/1591666).
+This comes at a time where the market share of IE11 has [dropped to below 1%](https://caniuse.com/usage-table).
+
+One of the major goals of the convergence project in Fluent v9, is to address past drawbacks and issues with both Fabric v8 and Northstar v0 and
+improve the experience for web developers at Microsoft. This has given an opportunity for the team to take a step back, look at the
+fundamentals and rethink our support for IE11.
+
+In this RFC we will explain our motivations for this decision and finally formalize this decision in writing.
+
+## Detailed Design or Proposal
+
+### Keep calm and carry on
+
+Our partners and products in M365 will either adhere to the August 2021 deadline or receive a 6 month extension for IE11 support.
+Since Fluent v9 is still in alpha release, and is planned to be at beta at August 2021, we will be ready for early adopters.
+Until that deadline, users that require full support for IE11 can do so with the existing versions of `@fluentui/react` and
+`@fluentui/react-northstar` that do provide support.
+
+## Motivation
+
+### Reduced transpilation - (perf + bundlesize improvements)
+
+The possibility to use native ES2015/ES6 features without the risks to breaking browser compatibility, we can reduce the
+transpilation of IE11 features that results in more code. This can result in perf and bundle size gains, and tslib
+will no longer be needed as a hard dependency to emulate the missing features of modern javascript.
```

Comment:
Small thing, I see what you mean but the sentence as written doesn't quite make sense (I think it's a little more important in this case because more people may read this than most of our RFCs). Maybe something like this?
```suggestion
The ability to use ES2015/ES6 features without the transpilation required for IE11 will result in perf and bundle size improvements, 
and tslib will no longer be needed as a dependency to emulate the missing features of modern javascript.
```
It's also not clear to me what compilation target results in tslib/helpers no longer being needed, and whether it's possible that helpers could be needed again in the future as new syntax is introduced.

Reply:
It's possible that newer language features in the future might have an effect, but since our support moves to evergreen browsers, it will definitely be a major departure from tslib without IE11.

Will reflect this in the RFC